src/_guides/language/spec.md

```diff
@@ -1,73 +1,29 @@
 ---
 layout: default
 title: "Dart Language Specification"
-description: "The Dart language specification and proposed changes."
+description: "The Dart language specification."
 ---
 
-Download the _Dart Programming Language Specification_ from
-the Ecma website:
-
-* <a href="http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-408.pdf"
-   target="_blank">Dart Programming Language Specification, 4<sup>th</sup> Edition</a>
-
+The following information talks about the formal Dart language specification.
 For a gentler introduction to the Dart language, see the
 [Dart language tour](/guides/language/language-tour).
 
-You can find previous editions of the specification at
-[Standard ECMA-408](http://www.ecma-international.org/publications/standards/Ecma-408-arch.htm).
-
-
-## Dart 2 changes
+## Dart 2
 
 Dart 2 has changed the Dart language in many ways,
 some of which are not backward-compatible.
-For information about changes and implementation status,
-see the following resources:
-
-* [Dart 2 Updates](/dart-2)
-* [Dart Language and Library Newsletters](https://github.com/dart-lang/sdk/tree/master/docs/newsletter#dart-language-and-library-newsletters)
-* [Language specification source file (TeX format)](https://github.com/dart-lang/sdk/blob/master/docs/language/dartLangSpec.tex)
-* [Informal specifications](https://github.com/dart-lang/sdk/tree/master/docs/language/informal)
-
-
-## Changes in the 4<sup>th</sup> edition
-
-The 4<sup>th</sup> edition of the specification has fewer restrictions on mixins.
-The restrictions on mixins up until this edition were:
+For additional details, please see the [Dart 2 changelog](https://github.com/dart-lang/sdk/blob/master/CHANGELOG.md#200---2018-08-07).
```

Comment:
I believe that it would be better to direct readers to `/dart-2` since that page describes some of the some of the details and contains links to yet further information (including the changelog). How about:

> For details, see the [Dart 2](/dart-2) page.